src/source-pcap.c

```diff
@@ -641,3 +705,223 @@ void PcapTranslateIPToDevice(char *pcap_dev, size_t len)
 
     pcap_freealldevs(alldevsp);
 }
+
+#ifdef UNITTESTS
```

Comment:
for new tests we put them in src/tests/source-pcap.c and include the file directly. See `stream-tcp-list.c tests/stream-tcp-list.c` as an example.

Reply:
Thanks for that! I copied the wrong example. ;) Will fix and open a new version.